src/mixins/map-mixin.js

```diff
@@ -277,8 +277,9 @@ export default function mapMixin(
     mapboxlogo.target = "_blank"
     mapboxlogo.innerHTML = "Mapbox"
 
-    const existingLogo = document.getElementsByClassName('mapbox-maplogo')
-    if(existingLogo.length  ) {
+    const thisMapContainer = _map._container
+    const existingLogo = thisMapContainer.querySelectorAll('.mapbox-maplogo')
```

Comment:
Seems like we could easily just use `querySelector` instead here, but unlikely to matter

Reply:
Changed because `querySelector` is better style